### DISCUSSION TITLE
How do I create a user with Docker-compose?

### DISCUSSION THREAD

**Discussion by gerroon** [ArchiveBox/ArchiveBox#1285] [Q&A] [answered]
Hi

I login to the container and run the command to create the user but I am getting some errors. Any ideas? It automatically logins as root.

```
docker exec -ti archivebox_docker_archivebox_1 bash

root@53628c7b1dfd:/data# archivebox manage createsuperuser

[i] [2023-12-15 02:27:31] ArchiveBox v0.7.1+editable: archivebox manage createsuperuser
    > /data

[!] ArchiveBox should never be run as root!
    For more information, see the security overview documentation:
        https://github.com/ArchiveBox/ArchiveBox/wiki/Security-Overview#do-not-run-as-root
root@53628c7b1dfd:/data#
root@53628c7b1dfd:/data# archivebox manage createsuperuser



```

**Comment by pirate** [comment, marked as answer]
Do not use exec, use `archivebox run manage create superuser`. 

I will add a hint to the warning message in a future release since this is a common mistake. 

Edit: done https://github.com/ArchiveBox/ArchiveBox/pull/1297/files#diff-b627cf6681c8f031fa16213db7f26ff256144856eea7c614691c357728317dc6R1129:~:text=When%20using%20Docker%2C%20you%20must%20run%20commands%20with

**Reply by gerroon** [reply]
Awesome, thank you!
